app/queries/customer_invoices_query.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+class CustomerInvoicesQuery < BaseQuery
```

Comment:
Is there a reason to add a specific `CustomerInvoicesQuery`?

It seems we could use the `InvoicesQuery` by adapting the possible filters/query arguments.
What do you think?

Reply:
hum yeah we could, just wondering about if the customer_id is passed or not, should we filter on customer ID or find customer.invoices as we do today.

Wondering if there is perf issue with the first approach. Let's discuss about it today

Reply:
Updated the code to have one query used for both  invoice resolvers